test/support/utils.rb

```diff
@@ -29,15 +31,13 @@ def debug_code(program, boot_options: '-r debug/run', remote: true, &block)
         debug_on_unix_domain_socket
         debug_on_tcpip
       end
-
-      check_line_num!(program)
     end
 
-    DEBUG_MODE = false
+    ENV['RUBY_DEBUG_TEST_DEBUG_MODE'] ||= 'false'
     ASK_CMD = %w[quit delete kill undisplay]
 
     def debug_print msg
-      print msg if DEBUG_MODE
+      print msg if ENV['RUBY_DEBUG_TEST_DEBUG_MODE']
```

Comment:
shouldn't this be 

```ruby
print msg if ENV['RUBY_DEBUG_TEST_DEBUG_MODE'] != "false"
```
because it's a string instead of boolean?

Reply:
Thanks! That was my mistake.